torch_xla/core/xla_model.py

```diff
@@ -19,6 +19,8 @@
 import torch_xla.utils.utils as xu
 import torch_xla.utils.closures as xc
 import os
+from torch_xla.experimental.deprecation import deprecated
+import torch_xla._internal.utils as iutils
```

Comment:
I'm not sure about `iutils` as the name of this module. IMO `_utils` is more idiomatic.  Another option is to move all of `torch_xla.utils.utils` to `_internal` and continue to use `xu` as the name for our internal utilities. We can put that off to a future PR.

Reply:
Use _utils for now. Will relocate later.